Remove modules with !important flags to pass AMP validation

mic-mart opened the pull request:
Fixes #1  

Assuming you're ok with commenting out `visibility` and `background-size` modules.

Commit:
Use more verbose `--minify` flag for Tachyons CLI (see https://github.com/tachyons-css/cli/issues/19)

Commit:
Comment out `visibility` and `background-size` modules that are using illegal `!important` flag.

Commit:
Add `.gitignore` from main Tachyons repo so that `node_modules` aren't included in repo.

Commit:
Size changed due to removal of two modules.

johno commented:
Thanks for the PR! 💟 

Rather than commenting out the modules, maybe we should fully remove them? If it doesn't pass validation we may as well nuke it entirely from the amp codebase.

Commit:
Remove visibility and background-size modules completely due to use of illegal !important flag.

mic-mart commented:
@johno I removed them. Let me know if you want to change anything else. I also switched the minify flag from `-m` to `--minify` since the former wasn't working.

johno commented:
Thanks!
